scripts/ci/kubernetes/ci_run_helm_testing.sh

```diff
@@ -20,6 +20,8 @@ echo "Running helm tests"
 
 chart_directory="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )/../../../chart/"
 
+pytest chart/tests
```

Comment:
I am not sure this is the correct way to run these tests.  I think we should use Docker to ensure a uniform development environment.